addons/pos_sale/static/src/js/OrderManagementScreen/SaleOrderManagementScreen.js

```diff
@@ -202,7 +202,7 @@ odoo.define('pos_sale.SaleOrderManagementScreen', function (require) {
                         };
                     }
                     let down_payment_product = this.env.pos.db.get_product_by_id(this.env.pos.config.down_payment_product_id[0])
-                    let down_payment_tax = this.env.pos.taxes_by_id[down_payment_product.taxes_id]
+                    let down_payment_tax = this.env.pos.taxes_by_id[down_payment_product.taxes_id] || 0 ;
```

Comment:
I think that in the case where there is no tax on the down payment we want to take on the amount_total.

So maybe we should do sometihing like:

let down_payment_tax = this.env.pos.taxes_by_id[down_payment_product.taxes_id] || false ;
let downpayment;
if (down_payment_tax) {
  down_payment = down_payment_tax.price_include ? sale_order.amount_total : sale_order.amount_untaxed;
}
else{
  down_payment = sale_order.amount_total;
}